projects/admin/src/app/record/detail-view/budget-detail-view/budget-detail-view.component.html

```diff
@@ -0,0 +1,36 @@
+<!--
+  RERO ILS UI
+   Copyright (C) 2019 RERO
+  
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU Affero General Public License as published by
+   the Free Software Foundation, version 3 of the License.
+  
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+   GNU Affero General Public License for more details.
+  
+   You should have received a copy of the GNU Affero General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<ng-container *ngIf="record$ | async as record">
+  <header class="row">
+    <h1 class="mb-1">{{ record.metadata.name }}</h1>
+  </header>
+  <section class="row">
+    <!-- Total -->
+    <dt class="col-sm-3 offset-sm-0 offset-md-0 pl-0">
+      {{ 'Total' | translate }}:
```

Comment:
use translate a tag field with a css class to add `:`

Reply:
done